### PR TITLE
Feature/boomer amg for elasticity

### PR DESCRIPTION
**Pull request by rrsettgast**

### CODE REVIEW

File: src/ArrayOfArraysView.hpp [hunk 1]
```diff
@@ -558,6 +579,12 @@ class ArrayOfArraysView
     m_offsets.move( space, touch );
   }
 
+
+  void moveValues( MemorySpace const space, bool touch=true ) const
```
Comment (corbett5) [inline src/ArrayOfArraysView.hpp:583]: What did you need this for that `move` didn't accomplish?

**Review by corbett5 — APPROVED** [review]

File: src/limits.hpp [hunk 2]
```diff
@@ -117,7 +117,8 @@ integerConversion( INPUT input )
   static_assert( std::is_integral< INPUT >::value, "INPUT must be an integral type." );
   static_assert( std::is_integral< OUTPUT >::value, "OUTPUT must be an integral type." );
 
-  return OUTPUT{ input };
+//  return OUTPUT{ input };
+  return static_cast<OUTPUT>(input);
```
Comment (corbett5) [inline src/limits.hpp:121]: The braced initialization broke something?

**Comment by corbett5** [comment]
Oops I just saw this was a WIP, but looks good to me!

File: src/ArrayOfArraysView.hpp (hunk 1 above)
Comment (rrsettgast) [reply src/ArrayOfArraysView.hpp:583]: Oh....It was to just move the "values" and not any of the other stuff. I think it is unnecessary now. I will probably remove this.

File: src/limits.hpp (hunk 2 above)
Comment (rrsettgast) [reply src/limits.hpp:121]: yes...but I think I fixed it elsewhere. I will try to revert this.

Comment (rrsettgast) [reply src/limits.hpp:121]: there was an integer narrowing error.

**Pull request merged**